cves/CVE-2002-1593.yml

```diff
@@ -131,12 +151,15 @@ major_events:
 
     The event doesn't need to be directly related to this vulnerability, rather,
     we want to capture what the development team was dealing with at the time.
-  answer:
-  events:
-    - name:
-      date:
-    - name:
-      date:
+  answer: There were no specific major events
+  events: |
+    There was no changes to this problem or really any acknowledgement
+    except the fact that it was announced. There were no major exploits with
+    this
+    vulnerability. The only problem it had caused was denial of service issues.
+    The bug was fixed in the Apache 2.0.42 release and that release was not
+    backwards compatabile which shows how important the fixes and features in
```

Comment:
What do you mean by not backwards compatible? That a patch wasn't released for previous versions or that they made abi breaking changes?

Also, compatabile should probably be fixed to compatible.

Reply:
The Apache release was not backwards compatible, so systems using older systems would not properly integrate with Apache 2.0.42.